packages/wrangler/src/index.tsx

```diff
@@ -751,8 +753,7 @@ export async function main(argv: string[]): Promise<void> {
       }
 
       const upstreamProtocol =
-        (args["upstream-protocol"] as "http" | "https") ||
-        config.dev.upstream_protocol;
+        args["upstream-protocol"] || config.dev.upstream_protocol;
```

Comment:
Nice. I wonder why we needed the cast before?

Reply:
shrug emoji